app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -95,73 +103,15 @@ If you don’t want to come to England to study you can apply to train for inter
 
 For teachers who qualified outside the UK, as well as those with teaching experience, [other routes to qualified teacher status (QTS)](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk) are also open.
 
-## Need to know
-
-When you plan your training, bear in mind that:
-
-* international candidates can expect to pay an average of £14,765, plus living costs, to train 
-* depending on your immigration status and the subject you train to teach, you may get some financial support
-* training lasts one academic year full-time, or longer part-time
-* you can apply for paid teaching jobs as soon as you successfully complete your training
-* you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher
-* you can update, extend or switch your visa to live and work in the UK long-term
-
-
-
-## Apply for teacher training
-
-We can help you make an application for teacher training in England. Sign up for a teacher training adviser to get help right away and throughout the application process.
+## Check your qualifications and get help
 
-### Get support and advice
+Before you apply, you must [check your qualifications](/non-uk-teachers/non-uk-qualifications). If you gained your qualifications outside the UK, we can help you check they meet the required standard in England. 
 
-For help applying, you can:
+For help checking your qualifications and completing your application for teacher training, you can:
 
-* sign up to [get a teacher training adviser](/teacher-training-adviser/sign_up/identity) – you’ll be asked for your phone number, email and location so a dedicated adviser can call you back in your time zone
 * call +44 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time) except on [bank holidays](https://www.gov.uk/bank-holidays) in England - calls will be charged at your standard rate
```

Comment:
@laurahermionetennant92 on the other page live chat is first and then the phone number. So it might be worth switching them round here so they're consistent?

Reply:
yes good point